src/base/USong.pas

```diff
@@ -680,17 +667,7 @@ function TSong.LoadOpenedSong(SongFile: TTextFileStream; FileNamePath: IPath; Du
           Param1 := ParseLyricIntParam(CurLine, LinePos);
           if self.Relative then
             Param2 := ParseLyricIntParam(CurLine, LinePos); // read one more data for relative system
-
-          // new sentence
```

Comment:
doesn't removing this break the P0 feature?

Reply:
I made a custom txt that used P3. Is P0 supposed to the same as P3=P1+P2? From the code it read as if only P1, P2 and P3 were ever read, but this should be easy enough to verify by duplicating my testfile with P3 to a file that uses P0 instead. I'll reply back once I figured out what it did without and with this PR.

Reply:
> doesn't removing this break the P0 feature?

Can you elaborate? What's the P0 feature? Never heard of or seen it anywhere.

Reply:
I checked with a P0 file, it doesn't load in master and also doesn't load in this branch, logged error is the same as the one in the PR description

@basisbit from what I understood of it:
- using P3 for "all players" was definitely a bad idea
- I don't think the debate about whether to use `P0` vs `P1+P2+P3` vs `P1,P2,P3` vs `P1/P2/P3` as whatever format to mean "all players" was ever closed. There was also a request that whatever ends up being chosen, it should have proper editor support (which USDX' P3 handling never had, it just outputted separate sections for P1 and P2)

By getting this in we can at least start reworking the USDX internals to support a generic number of tracks, and actually do some experiments with what kind of UI we need to select who sings what, what kind of theme elements we need for it.